doc/website/release-notes/iceoryx-unreleased.md

```diff
@@ -19,17 +19,39 @@
   - Create separate error enum for each module
 - Use `GTEST_FAIL` and `GTEST_SUCCEED` instead of `FAIL` and `SUCCEED` [\#1072](https://github.com/eclipse-iceoryx/iceoryx/issues/1072)
 - posix wrapper `SharedMemoryObject` is silent on success [\#971](https://github.com/eclipse-iceoryx/iceoryx/issues/971)
+- posix wrapper `SharedMemoryObject` uses builder pattern instead of creation [\#1036](https://github.com/eclipse-iceoryx/iceoryx/issues/1036)
+  - Builder pattern extracted from `helplets.hpp` into `design_pattern/builder_pattern.hpp`
```

Comment:
This is not really a subtask of the item above. I think it's better to either follow the suggestion of @elBoberido or have both items on the same level.

Reply:
The reason why I stated this explicitly was that this leads to a change in the API since users now have to include a different header when they want to use the builder pattern. But I can remove this.